src/pages/Markets/MarketsHeader.tsx

```diff
@@ -43,15 +38,6 @@ export const MarketsHeader = () => {
       history.location.pathname.match(/\/markets\/category\/(?<category>[\w]+)/)?.groups?.category,
     [history.location.pathname],
   )
-  const headingCopy = useMemo(() => {
-    if (maybeCategory) return `markets.categories.${maybeCategory}.title`
-    return 'navBar.markets'
-  }, [maybeCategory])
-
-  const subtitleCopy = useMemo(() => {
-    if (maybeCategory) return `markets.categories.${maybeCategory}.subtitle`
-    return 'markets.marketsBody'
-  }, [maybeCategory])
```

Comment:
No need to be programmatic here as we don't display the bigly header while in the context of a category, these have their own